### ISSUE TITLE
12th CPU 12700K GPU or unrecognized 

### ISSUE THREAD

**Issue by keacefull** [fabianishere/pve-edge-kernel#248]
12700K GPU or unrecognized，5.16 Not supported？
![image](https://user-images.githubusercontent.com/8847289/156334444-b2f64803-aba9-43b9-94e4-668721123b0c.png)

**Comment by fabianishere** [comment]
I don't know why it is not working. The driver is included in the 5.16 build.

**Comment by chchia** [comment]
it is using vfio-pci, you must have enabled the blacklist.

mine 12400 CPU

```
*-display
       description: VGA compatible controller
       product: Intel Corporation
       vendor: Intel Corporation
       physical id: 2
       bus info: pci@0000:00:02.0
       version: 0c
       width: 64 bits
       clock: 33MHz
       capabilities: pciexpress msi pm vga_controller bus_master cap_list rom
       configuration: driver=i915 latency=0
       resources: iomemory:650-64f iomemory:400-3ff irq:158 memory:651a000000-651affffff memory:4000000000-400fffffff ioport:8000(size=64) memory:c0000-dffff memory:6513000000-6519ffffff memory:6420000000-64ffffffff
```
